sdk/batch/microsoft-azure-batch/CHANGELOG.md

```diff
@@ -1,10 +1,16 @@
 # Release History
 
+## 8.1.1 (Unreleased)
```

Comment:
Whichever version we plan to release we'll want to add a date to it

Reply:
Should I use today's date?